pkg/apis/fleet.cattle.io/v1alpha1/bundle.go

```diff
@@ -237,6 +237,9 @@ type HelmOptions struct {
 
 	// Atomic sets the --atomic flag when Helm is performing an upgrade
 	Atomic bool `json:"atomic,omitempty"`
+
+	// DisablePreprocess deactivates fleet YAML templating
+	DisablePreprocess bool `json:"disablePreprocess,omitempty"`
```

Comment:
This seems to make everything more complicated. Why would we want to disable it, for performance, security?